Fix ax25 header file include problems
--- a/aprsdigi.c
+++ b/aprsdigi.c
@@ -55,9 +55,7 @@
 #include <time.h>
 #include <signal.h>
 #include <errno.h>
-#include <netax25/ax25.h>
-#include <netax25/axlib.h>
-#include <netax25/axconfig.h>
+#include <linux/ax25.h>
 #ifndef AX25_MTU
 #define AX25_MTU 256
 #endif
--- a/libax25ext.c
+++ b/libax25ext.c
@@ -9,7 +9,6 @@
 
 #include <netax25/ax25.h>
 #include <netrose/rose.h>
-#include <netax25/axlib.h>
 #include "libax25ext.h"
 #include <string.h>
 
--- a/aprsmon.c
+++ b/aprsmon.c
@@ -61,9 +61,7 @@
 #include <time.h>
 #include <signal.h>
 
-#include <netax25/ax25.h>
-#include <netax25/axlib.h>
-#include <netax25/axconfig.h>
+#include <linux/ax25.h>
 
 #ifndef HAVE_LIBAX25_EXTENSIONS
 #include "libax25ext.h"
--- a/aprsshm.c
+++ b/aprsshm.c
@@ -44,9 +44,6 @@
 #include <ctype.h>
 #include <time.h>
 
-#include <netax25/ax25.h>
-
-#include <netax25/axconfig.h>
 #include "aprsshm.h"
 
 #define MAXPPM 10
--- a/testmcast.c
+++ b/testmcast.c
@@ -16,9 +16,7 @@
 #include <time.h>
 #include <signal.h>
 #include <errno.h>
-#include <netax25/ax25.h>
-#include <netax25/axlib.h>
-#include <netax25/axconfig.h>
+#include <linux/ax25.h>
 #ifndef AX25_MTU
 #define AX25_MTU 256
 #endif
--- a/testparse.c
+++ b/testparse.c
@@ -16,9 +16,7 @@
 #include <time.h>
 #include <signal.h>
 #include <errno.h>
-#include <netax25/ax25.h>
-#include <netax25/axlib.h>
-#include <netax25/axconfig.h>
+#include <linux/ax25.h>
 #include <netdb.h>
 #include "libax25ext.h"
 static void print_it(FILE *f,
